### PR TITLE
Add foreman-ec2 and recomend it for install with compute

### PR DESCRIPTION
**Pull request by GregSutcliffe**
Also drops the old conflicts on foreman-fog from the 1.2 era.

### CODE REVIEW

File: debian/precise/foreman/control [hunk 1]
```diff
@@ -88,16 +88,25 @@ Package: foreman-compute
 Architecture: all
 Section: web
 Priority: extra
-Replaces: foreman-fog
-Provides: foreman-fog
-Conflicts: foreman-fog
+Recommends: foreman-ec2
 Depends: foreman, libxml2-dev, libxslt1-dev, ${misc:Depends}
-Description: metapackage providing fog dependencies for Foreman (for Amazon EC2 support)
```
Comment (domcleal) [inline debian/precise/foreman/control]: The RPMs say it includes OpenStack and Rackspace support, perhaps say that instead?

File: debian/precise/foreman/control.orig [hunk 2]
```diff
@@ -0,0 +1,188 @@
+Source: foreman
```
Comment (domcleal) [inline debian/precise/foreman/control.orig]: No .orig please :)

**Comment by domcleal** [comment]
If you can run a scratch build once that's fixed, it'd be appreciated, I'll test it.

**Comment by domcleal** [comment]
@GregSutcliffe?

**Comment by domcleal** [comment]
Tidied up, added missing .postinst files and merged as 936fcd3.  Thanks @GregSutcliffe!

**Comment by GregSutcliffe** [comment]
thanks @domcleal this slipped off my radar, and some time off didn't help. Cleanup much appreciated :)

**Pull request closed without merging**